tests/wpt/metadata-layout-2020/referrer-policy/gen/iframe.http-rp/unset/script-tag.http.html.ini

```diff
@@ -0,0 +1,20 @@
+[script-tag.http.html]
+  expected: TIMEOUT
+  [Referrer Policy: Expects stripped-referrer for script-tag to same-https origin and keep-origin redirection from http context.]
+    expected: NOTRUN
+
+  [Referrer Policy: Expects stripped-referrer for script-tag to same-http origin and no-redirect redirection from http context.]
+    expected: NOTRUN
+
+  [Referrer Policy: Expects stripped-referrer for script-tag to same-http origin and swap-origin redirection from http context.]
+    expected: NOTRUN
+
+  [Referrer Policy: Expects stripped-referrer for script-tag to same-http origin and keep-origin redirection from http context.]
+    expected: TIMEOUT
+
+  [Referrer Policy: Expects stripped-referrer for script-tag to same-https origin and swap-origin redirection from http context.]
+    expected: NOTRUN
+
+  [Referrer Policy: Expects stripped-referrer for script-tag to same-https origin and no-redirect redirection from http context.]
+    expected: NOTRUN
```

Comment:
Both layout-2013 Linux and macOS got PASS results so I think it was an intermittent TIMEOUT 🤔?

```
  ▶ OK [expected TIMEOUT] /referrer-policy/gen/iframe.http-rp/unset/script-tag.http.html
  ▶ Unexpected subtest result in /referrer-policy/gen/iframe.http-rp/unset/script-tag.http.html:
  └ PASS [expected TIMEOUT] Referrer Policy: Expects stripped-referrer for script-tag to same-http origin and keep-origin redirection from http context.
  ▶ Unexpected subtest result in /referrer-policy/gen/iframe.http-rp/unset/script-tag.http.html:
  └ PASS [expected NOTRUN] Referrer Policy: Expects stripped-referrer for script-tag to same-http origin and no-redirect redirection from http context.
  ▶ Unexpected subtest result in /referrer-policy/gen/iframe.http-rp/unset/script-tag.http.html:
  └ PASS [expected NOTRUN] Referrer Policy: Expects stripped-referrer for script-tag to same-http origin and swap-origin redirection from http context.
  ▶ Unexpected subtest result in /referrer-policy/gen/iframe.http-rp/unset/script-tag.http.html:
  └ PASS [expected NOTRUN] Referrer Policy: Expects stripped-referrer for script-tag to same-https origin and keep-origin redirection from http context.
  ▶ Unexpected subtest result in /referrer-policy/gen/iframe.http-rp/unset/script-tag.http.html:
  └ PASS [expected NOTRUN] Referrer Policy: Expects stripped-referrer for script-tag to same-https origin and no-redirect redirection from http context.
  ▶ Unexpected subtest result in /referrer-policy/gen/iframe.http-rp/unset/script-tag.http.html:
  └ PASS [expected NOTRUN] Referrer Policy: Expects stripped-referrer for script-tag to same-https origin and swap-origin redirection from http context.
```